### ISSUE TITLE
Add caps for export of referrals and settings

### ISSUE THREAD

**Issue by ramiabraham** [AffiliateWP/AffiliateWP#1452]
PR: #1480 

The AffiliateWP caps defined presently do not allow exporting of referrals or settings on the `Tools` page for non-admins, in any case.

Given the custom function below, all users in the array will be permitted to view `Affiliates` &rarr; `Tools`, as well as export affiliates, but cannot export anything else.

```php
function affwp_custom_add_affiliate_manager_caps() {

    $user_ids = array( 2, 5, 7, 12 );

    foreach ( $user_ids as $user_id ) {
        $user = new WP_User( $user_id );
        $user->add_cap( 'view_affiliate_reports' );
        // Exporting affiliate data works fine
        $user->add_cap( 'export_affiliate_data' );
        $user->add_cap( 'manage_affiliate_options' );
        $user->add_cap( 'manage_affiliates' );
        $user->add_cap( 'manage_referrals' );
        $user->add_cap( 'manage_visits' );
        $user->add_cap( 'manage_creatives' );

        // Does not yet exist
        $user->add_cap( 'export_referrals_data' );
        // Does not yet exist
        $user->add_cap( 'export_visits_data' );
    }
}
add_action( 'admin_init', 'affwp_custom_add_affiliate_manager_caps');
```

_Related: https://secure.helpscout.net/conversation/226466403/37978/?folderId=205529_

**Comment by ramiabraham** [comment]
Fixed in #1480

**Comment by ramiabraham** [comment]
Here's a quick plugin to test this; change user id array to any non-admin user:

https://gist.github.com/ramiabraham/5ca12e9f3098df053535e1e6bd96c3d9

**Comment by DrewAPicture** [comment]
Works in my testing.

**Comment by DrewAPicture** [comment]
Tests passed, merged #1480.
